pyproject.toml

```
@@ -49,7 +49,8 @@ classifiers = [
 ]
 
 # Python dependencies required for use.
-requires-python = ">=3.10"
+# coremltools has issue with python 3.13, see https://github.com/apple/coremltools/issues/2487 
+requires-python = ">=3.10,<=3.12"
```

I guess we should change this to sub 13
```suggestion
requires-python = ">=3.10,<3.13"
```